spec/models/edition_spec.rb

```diff
@@ -82,8 +84,10 @@
   end
 
   describe "#history_mode?" do
+    before { populate_default_government_bulk_data }
+
     it "returns true when political and associated with a previous government" do
-      edition = build(:edition, :political, :past_government)
+      edition = build(:edition, :political, government: past_government)
```

Comment:
No, unfortunately not. That's how I had it set up at first and then got caught out by the FactoryBot linter. Reflecting on that I thought it was pretty weird to have a factory have an external dependency.

The bummer of moving the `populate_default_government_bulk_data` into the factory is it will be destructive of anything already set, so we'd then need a `if empty` condition and the complexity grows.